example/cajita_tutorial/11_semi_structured_solver_hypre/hypre_semi_structured_solver_example.cpp

```diff
@@ -0,0 +1,189 @@
+/****************************************************************************
+ * Copyright (c) 2018-2022 by the Cabana authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the Cabana library. Cabana is distributed under a   *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <Cajita.hpp>
+
+#include <Kokkos_Core.hpp>
+
+#include <mpi.h>
+
+#include <array>
+#include <vector>
+
+//---------------------------------------------------------------------------//
+// HYPRE Structured Solver Example
+//---------------------------------------------------------------------------//
+void hypreSemiStructuredSolverExample()
```

Comment:
I'm not sure it makes sense to keep this example - we have the original solver and the multi-variate